core/templates/domain/story/story-node.model.ts

```diff
@@ -29,9 +29,9 @@ export interface StoryNodeBackendDict {
   'acquired_skill_ids': string[];
   'outline': string;
   'outline_is_finalized': boolean;
-  'exploration_id': string;
-  'thumbnail_bg_color': string;
-  'thumbnail_filename': string;
+  'exploration_id': string | null;
+  'thumbnail_bg_color': string | null;
+  'thumbnail_filename': string | null;
```

Comment:
ditto, why `null` abd would it be possible to do it without `null`?

Reply:
In the last function in `story-node.model.ts`, a new StoryNode is being returned by explicitly passing `null` value to the last three parameters.
```typescript
  static createFromIdAndTitle(nodeId: string, title: string): StoryNode {
    return new StoryNode(
      nodeId, title, '', [], [], [], '', false, null,
      null, null);
  }

```
They are being treated different from the arguments passed for `description`(3rd) and `outline`(7th) which were also of type `string`

Reply:
Okay, makes sense, you can do this I think:
```suggestion
  'exploration_id'?: string;
  'thumbnail_bg_color'?: string;
  'thumbnail_filename'?: string;
```

Reply:
I believe that doing this will convert the type from `string` to `string | undefined` and not `string | null`. We will not be able to pass `null` as a value for these properties.

Reply:
Okay, makes sense.